### PR TITLE
feat: import wallet to store

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-ui/src/containers/MainContent/state.ts [hunk 1]
```diff
@@ -2,9 +2,14 @@ import { CapacityUnit } from '../../utils/const'
 
 export const initState = {
   tempWallet: {
```
Comment (Keith-CY) [inline packages/neuron-ui/src/containers/MainContent/state.ts]: It seems that `tempWallet` have the same functionality as `createWallet` in the state.

File: packages/neuron-wallet/src/channel/index.ts [hunk 2]
```diff
@@ -165,12 +168,47 @@ export class Listeners {
...
+          }
+          if (storedKeystore) {
+            walletStore.saveWallet(walletName, storedKeystore)
+            e.sender.send(Channel.ImportWallet, {
+              status: ResponseStatus.Success,
+              result: true,
+            })
+          } else {
+            e.sender.send(Channel.ImportWallet, {
+              status: ResponseStatus.Success,
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/channel/index.ts]: ResponseState.Fail

File: packages/neuron-wallet/src/channel/index.ts [hunk 3]
```diff
@@ -165,12 +168,47 @@ export class Listeners {
...
+              result: false,
+              msg: 'Error',
+            })
+          }
+        } catch (error) {
+          e.sender.send(Channel.ImportWallet, {
+            status: ResponseStatus.Success,
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/channel/index.ts]: ResponseState.Fail

Comment (BaojunCZ) [reply packages/neuron-wallet/src/channel/index.ts]: ok

Comment (Keith-CY) [reply packages/neuron-wallet/src/channel/index.ts]: I've fix it in another PR.

Comment (BaojunCZ) [reply packages/neuron-wallet/src/channel/index.ts]: ok

**Pull request merged**